modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java

```diff
@@ -101,9 +107,31 @@ public class UserLocalServiceTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@BeforeClass
+	public static void setUpClass() {
+		_noticeableExecutorService = _portalExecutorManager.getPortalExecutor(
+			UserLocalServiceTest.class.getName());
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+		_noticeableExecutorService.shutdown();
+	}
+
 	@Test
 	public void testAuthenticateByEmailAddress() throws Exception {
-		User user = UserTestUtil.addUser();
+		NoticeableFuture<User> userNoticeableFuture =
+			_noticeableExecutorService.submit(
+				new Callable<User>() {
+
+					@Override
+					public User call() throws Exception {
+						return UserTestUtil.addUser();
+					}
+
+				});
+
+		User user = userNoticeableFuture.get();
```

Comment:
@ling-alan-huang, I tried to use a lambda expression here like:
```
_noticeableExecutorService.submit(()-> UserTestUtil.addUser())
```
but the SF is converting this to a method reference call `UserTestUtil::addUser` causing a compile issue due to an ambiguity between the `UserTestUtil.addUser()` and the `UserTestUtil.addUser(boolean)`.

Thx

Reply:
I will have a look at it later.

Reply:
Hey @rafaprax Can you send me your changes that causing compile error?

Reply:
sure